New package: XAMAuxData v0.1.0

JuliaRegistrator opened the pull request:
- Registering package: XAMAuxData
- Repository: https://github.com/BioJulia/XAMAuxData.jl
- Created by: @jakobnissen
- Version: v0.1.0
- Commit: 72b1091562d16abb3a2e6241c05b41a291f93b4f
- Reviewed by: @jakobnissen
- Reference: https://github.com/BioJulia/XAMAuxData.jl/commit/72b1091562d16abb3a2e6241c05b41a291f93b4f#commitcomment-143807636
- Description: Read auxiliary data fields in SAM, GFA and PAF formats
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd6e42836f6b695ef56928f701e899e85695e8030d884b0bdd0a670eafb1b970286977b67629d7ba2e7f8c0560b7fd8c9da3edd7570d6950d98c9c6d500f96456382eadc8e805b1fd6e09d4891644039a7582d44dc61f92fd866e46b01ad2cf716c2b816e06e49d5ad8c2f20a20f654ddfda88df4f4fad6dd57da16f8b24c5a34087c5144e312190f69a4cec7c6f354496153551b4b6ecfed4525b149493e85fb8159dd9192e4a0021909c21f61ba377d42 -->

jakobnissen commented:
[noblock] this currently only works on Julia master, due to issues with the current 1.11-rc1. Hence it cannot autoinstall

JuliaRegistrator committed:
New package: XAMAuxData v0.1.0

UUID: e99d641e-1821-45d7-9150-ecb7bf333fe1
Repo: https://github.com/BioJulia/XAMAuxData.jl.git
Tree: 6530435fd8f4bdad37829a454b07e0184c51beb9

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
